projects/plugins/crm/tests/php/automation/tools/class-automation-faker.php

```diff
@@ -202,19 +203,21 @@ public function workflow_with_condition_action(): array {
 			'steps'        => array(
 				// Step 0
 				0 => array(
-					'slug'            => 'contact_status',
-					'class_name'      => Contact_Condition::class,
+					'slug'            => Contact_Condition::get_slug(),
+					'next_step_true'  => 1,
+					'next_step_false' => null,
 					'attributes'      => array(
 						'field'    => 'status',
 						'operator' => 'is',
 						'value'    => 'lead',
 					),
-					'next_step_true'  => 1,
-					'next_step_false' => null,
 				),
 				// Step 1
 				1 => array(
-					'slug' => 'dummy_step',
+					'slug'            => Dummy_Step::get_slug(),
+					'next_step_true'  => null,
+					'next_step_false' => null,
+					'attributes'      => array(),
```

Comment:
Updated dummy data so they more accurately represents what would be fetched by `Base_Step::to_array()` which makes direct comparisons in tests much easier.